js/repl/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
```

Comment:
In theory, the JS can be rebuilt entirely separately from the Jekyll build, if you can configure it to build it directly into the `_site` directory. That way, you'd only care about the Jekyll build if you touch the HTML. I wonder if Jekyll would get angry at us for touching the `_site` files directly though. Could try it and see 😛 

Yeah, Jekyll is slow... Maybe we should transition the site to something like Hugo, it's a lot faster.